examples/IBFE/explicit/ex0/example.cpp

```diff
@@ -531,7 +531,8 @@ main(int argc, char* argv[])
             System& X_system = equation_systems->get_system<System>(IBFEMethod::COORDS_SYSTEM_NAME);
             NumericVector<double>* X_vec = X_system.solution.get();
             NumericVector<double>* X_ghost_vec = X_system.current_local_solution.get();
-            X_vec->localize(*X_ghost_vec);
+            *X_ghost_vec = *X_vec;
```

Comment:
cannot check right now, but is this equivalent to what `copy_and_synch` does?

Reply:
It does! Should we just use that instead?

Reply:
Yes?